clusterDeployer.py

```diff
@@ -86,6 +86,13 @@ def setup_vm(h: host.Host, virsh_pool: VirshPool, cfg: dict, iso_path: str):
     return ret
 
 
+def run_cmd(h, cmd):
```

Comment:
create function in Host called
`run_or_die`
We can use this in general so I'd like to have this as a function on Host.